tests/config/install.sh

```diff
@@ -141,7 +141,25 @@ if [[ -n "$USE_DATABASE_ORDINARY" ]] && [[ "$USE_DATABASE_ORDINARY" -eq 1 ]]; th
 fi
 
 if [[ -n "$USE_S3_STORAGE_FOR_MERGE_TREE" ]] && [[ "$USE_S3_STORAGE_FOR_MERGE_TREE" -eq 1 ]]; then
-    ln -sf $SRC_PATH/config.d/s3_storage_policy_by_default.xml $DEST_SERVER_PATH/config.d/
+    object_key_types_options=("generate-suffix" "generate-full-key" "generate-template-key")
+    object_key_type="${object_key_types_options[0]}"
+
+    if [[ -n "$RANDOMIZE_OBJECT_KEY_TYPE" ]] && [[ "$RANDOMIZE_OBJECT_KEY_TYPE" -eq 1 ]]; then
+      object_key_type="${object_key_types_options[$(($RANDOM % ${#object_key_types_options[@]}))]}"
+    fi
+
+    case $object_key_type in
```

Comment:
this breaks stress test because we don't create `s3_storage_policy_by_default.xml` file but later on we have
```sh
sudo cat /etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml \
  | sed "s|<main><disk>s3</disk></main>|<main><disk>s3</disk></main><default><disk>default</disk></default>|" \
  > /etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml.tmp
mv /etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml.tmp /etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml
```
because the files doesn't exist we will create an empty config file making CH unable to start:
```
Poco::Exception. Code: 1000, e.code() = 0, Exception: Failed to merge config with '/etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml': SAXParseException: No element found in '/etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml', line 1 column 0 (version 24.1.1.1)
```

related error from script:
```
sudo cat /etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml
sed 's|<main><disk>s3</disk></main>|<main><disk>s3</disk></main><default><disk>default</disk></default>|'
cat: /etc/clickhouse-server/config.d/s3_storage_policy_by_default.xml: No such file or directory
```

Reply:
Thank you.
The fix is here
https://github.com/ClickHouse/ClickHouse/pull/58740